### PR TITLE
HCAP-376 Available participants for My Sites

### PR DESCRIPTION
**Pull request by arranfw**
https://freshworks.atlassian.net/browse/HCAP-376
- Rewrote the sites query because fields causes a lot of unexpected issues within MassiveJS
- Add "Hires" column to sites table

### CODE REVIEW

**Review by fw-noel — DISMISSED** [review]

**Review by alva-fresh — COMMENTED** [review]
Looks good! Just one question - on the client side we're using `parseInt(siteId)`, do you think we still need it?

**Comment by arranfw** [comment]
> Looks good! Just one question - on the client side we're using `parseInt(siteId)`, do you think we still need it?

Possibly not, let me take a look

File: server/services/employers.js [hunk 1]
```diff
@@ -68,18 +68,26 @@ const getSitesForUser = async (user) => {
...
-      'siteName',
-      'operatorName',
-      'city',
-      'healthAuthority',
-      'postalCode',
-      'allocation',
-    ],
-  });
+  const records = (
+    await dbClient.db[collections.EMPLOYER_SITES].findDoc(criteria, {
```
Comment (taraepp) [inline server/services/employers.js:72]: I'm confused- this is the same function as https://github.com/bcgov/hcap/pull/834/files, right? Is this PR out of date and should be closed? Or are you wanting the other changes?

Comment (arranfw) [reply server/services/employers.js]: Sorry #834 was branched off this one, they're merged together now

**Review by taraepp — APPROVED** [review]

**Review by fw-noel — CHANGES_REQUESTED** [review]
Please see last comment

**Review by fw-noel — APPROVED** [review]

**Review by alva-fresh — APPROVED** [review]
Looks good to me!

**Pull request merged**